torch/csrc/jit/codegen/cuda/index_reference_replay.cpp

```diff
@@ -253,7 +254,7 @@ TensorDomain* IndexReferenceReplay::computeReplay() {
     }
 
     // Create and return the reference.
-    auto domain = IrBuilder::create<TensorDomain>(
+    auto domain = SimplifyingIrBuilder::create<TensorDomain>(
```

Comment:
Does this do anything? I don't think SimplifyingIrBuilder does anything on TensorDomian, Split, Merge, or IterDomain right? I don't know if it did something on split/merge that it would be okay in this file.

Reply:
Ah, I think I just replaced all of `IrBuilder` to `SimplyfingIrBuilder`. It doesn't feel right for `create`. I'll revert the changes.